koku/masu/database/trino_sql/reporting_gcpcostentrylineitem_daily_summary.sql

```diff
@@ -37,7 +37,11 @@ SELECT uuid() as uuid,
     nullif(location_region, '') as region,
     json_extract_scalar(json_parse(system_labels), '$["compute.googleapis.com/machine_spec"]') as instance_type,
     max(usage_pricing_unit) as unit,
-    cast(sum(usage_amount_in_pricing_units) AS decimal(24,9)) as usage_amount,
+    CASE
+        WHEN max(usage_pricing_unit) = 'gibibyte month'
+        THEN cast(sum(usage_amount_in_pricing_units) * 1.074 AS decimal(24,9)) -- Convert to gigabyte
```

Comment:
We may not want to round here, since this will be at a daily aggregation. It could add up over time, we may want to use the exact conversion which would be: `1.073741824`